src/resources/help/en_US/relnotes3.1.0.html

```diff
@@ -74,6 +74,7 @@
   <li>Changing the opacity for a Pseudocolor plot no longer affects the glyph type of glyphed points.</li>
   <li>Corrected an issue with installing host profiles after deleting or moving the .visit folder while VisIt is running.</li>
   <li>Removed the visual artifacts of the Expressions window that were present on first draw.</li>
+  <li>Fixed a memory leak with Pseudocolor plots.</li>
```

Comment:
This should probably indicate the leak is in the *viewer* and its not specific to any plot type, either.